### PR TITLE
Portfolio Publishing to gh-pages

### PR DESCRIPTION
**Pull request by Bavon101**
- [ ] Simplified the CSS styling
- [ ] Created a portfolio live branch
- [ ] Added branch `portfolio` as the GH-pages branch
- [ ] Pushed the branch changes and by default the site built

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 > My new Portfolio.
 <!-- ![screenshot](./hello_microverse.png) -->
```
Comment (enkog) [inline README.md:7]: - [ ] [OPTIONAL] Please include an actual project name and description in your README file. This is so visitors of this repository can firsthand know what the repo is about.

**Review by enkog — CHANGES_REQUESTED** [review]
Hi @Bavon101 👋 , 

Good job on deploying your portfolio to Github Pages 👍 !
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [ ] Please ensure that this 'overflow' on your page is fixed since this will allow visitors to your portfolio to have a smooth transition around the site and will demonstrate your skill to build a responsive design.

image




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ]  Centering the text on the navigation, i.e. the nav-links, using a text-align or flex property and then use margins or paddings to adjust the spacing. This is due to the clear extra space between the nav-links for 'portfolio' and 'about.'

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AleksandarIvezic) [reply README.md:7]: [Optional] 
As the previous reviewer asked, you can include an actual project name and description on top of your README file. 
image

**Review by AleksandarIvezic — APPROVED** [review]
## Hi @Bavon101 👋🏼 , 

STATUS: Approved! ✔️ ✔️ ✔️ 

Great work on making the changes requested by a previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

gif

Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider fixing this overlapping in the Contact section. I believe that you would agree that it will look much better if the heading and paragraph don't display over each other. 😃 

image




Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bavon101** [comment]
> ## Hi @Bavon101 👋🏼 ,
> STATUS: Approved! ✔️ ✔️ ✔️
> 
> Great work on making the changes requested by a previous reviewer. 👏 Your project is complete! There is nothing else to say other than... it's time to merge it 🚀 Congratulations! 🎉
> 
> gif
> 
> Cheers and Happy coding!👏👏👏
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> You can also consider fixing this overlapping in the Contact section. I believe that you would agree that it will look much better if the heading and paragraph don't display over each other. 😃
> 
> image
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

I'm not able to replicate the overlapping issue, could you please share the screen size at which such an overlap occurs or if it's a different browser cause I use chrome for test cases.,

**Pull request merged**